### PR TITLE
Basic isa json functionalities

### PR DESCRIPTION
**Pull request by kdp-cloud**
- Separate functionality from CLI
- Written tests for the already existing parts.
- Added GitHub workflow for MARS CLI functionalities

### CODE REVIEW

**Comment by M-casado** [comment]
Hi there @kdp-cloud & @bedroesb - Just wanted to let you guys know that I'm currently a bit busy with other projects, and given that my work is externally funded and I expect EGA not to be a use-case, I may not be able to dedicate much of it to this project. I'll try to sneak in a few bits here and there, but long reviews like these may wait for a long time

**Comment by kdp-cloud** [comment]
Hi @M-casado,

I can completely relate to that. The main reason why I asked you as reviewer, is because I did some changes in your code and wanted to give you, as a courtesy, the opportunity to review it.

**Comment by M-casado** [comment]
@kdp-cloud - Sure, I noticed minimal changes in the code and the new tests, I'm happy for changes to be made on what I wrote previously :)

**Review by bedroesb — APPROVED** [review]

File: mars-cli/.python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.9
```
Comment (bedroesb) [inline mars-cli/.python-version:1]: I was just wondering what this file is?

**Review by bedroesb — COMMENTED** [review]
Looks good to me! Cli also works, after this we can start thinking about the other functionalities!

**Review by bedroesb — APPROVED** [review]

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
ok sorry for all the approvals, vscode was tricking me :)

**Pull request merged**